linux_os/guide/system/auditing/authconfig_config_files_symlinks/rule.yml

```diff
@@ -0,0 +1,56 @@
+documentation_complete: true
+
+prodtype: ol7
+
+title: 'Ensure system-auth and password-auth files are symbolic links pointing to system-auth-local and password-auth-local'
```

Comment:
Could you ensure lines no longer than 99 columns, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#all-files

Reply:
Addressed